docs/02-app/01-building-your-application/02-data-fetching/02-server-actions-and-mutations.mdx

```diff
@@ -0,0 +1,832 @@
+---
+title: Server Actions and Mutations
+nav_title: Server Actions and Mutations
+description: Learn how to handle form submissions and data mutations with Next.js.
+---
+
+Server Actions are **asynchronous functions** that are executed on the server. They can be used in Server and Client Components to handle form submissions and data mutations in Next.js applications.
+
+> **🎥 Watch:** Learn more about forms and mutations with the App Router → [YouTube (10 minutes)](https://youtu.be/dDpZfOQBMaU?si=cJZHlUu_jFhCzHUg).
+
+## Behavior
+
+- Server actions can be invoked using the `action` attribute in a [`<form>` element](#forms):
+  - Server Components support progressive enhancement by default, meaning the form will be submitted even if JavaScript hasn't loaded yet or is disabled.
```

Comment:
Do we need to make a not about components being in a Suspense boundary not working with JS disabled somewhere?

Reply:
I think we should mention this in the Suspense documentation as it applies to all code, I'll create a linear task 👍🏼